test/unit/plugins/url-titles.js

```diff
@@ -61,4 +61,29 @@ describe('url-titles', () => {
 
     client.say.should.have.been.calledWith('you', '>>> lol')
   })
+
+  it('should change newlines in titles to spaces', () => {
+    var request = (uri, handler) => {
+      handler(null, 200,
+          `<html><head><title>lol
+
+          who put all these newlines in?
+
+          How silly.
+          </title></head><body>lol</body></html>`
+          )
+    }
+
+    var client = {
+      addListener: (name, callback) => {
+        callback('you', 'me', 'message containing url https://google.com :D')
+      },
+      say: sinon.spy(),
+      nick: 'me'
+    }
+
+    require('../../../src/plugins/url-titles.js')(client, request)
+
+    client.say.should.have.been.calledWith('you', '>>> lol who put all these newlines in? How silly.')
```

Comment:
I was debating this as well. I didn't want to split it as I was a little worried that splitting it over new lines might actually introduce them into the test? I wasn't sure anyway. I appreciate it's a bit long, but I suppose that's what it's testing that the output will be. Suggestions welcomed!